blockchair/src/endpoint/balance.ts

```diff
@@ -0,0 +1,147 @@
+import objectPath from 'object-path'
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest } from '@chainlink/types'
+import { Config, DEFAULT_DATA_PATH, getBaseURL } from '../config'
+import { CoinType, ChainType } from '.'
+
+export const Name = 'balance'
+
+type Address = {
+  address: string
+  coin?: CoinType
+  chain?: ChainType
+  balance?: number
+  warning?: string
+}
+
+type AddressGroup = {
+  addresses: string[]
+  result: Address[]
+}
+
+type GroupedAddresses = {
+  [network: string]: AddressGroup
+}
```

Comment:
It's hard for me to understand what are we grouping and why do we need these extra types at this point.